wiki/People/Quality_Assurance_Team/Blog/en.md

```diff
@@ -0,0 +1,71 @@
+# Quality Assurance Team Blog
+
+The **[Quality Assurance Team Blog](http://osuqat.tumblr.com)**, commonly referred to as the QAT Blog, is a semi-regular blog written by members of the [Quality Assurance Team](/wiki/People/Quality_Assurance_Team). Since August of 2016, they have been discussing not only the activities of the QAT, but a variety of other topics spanning osu! gameplay, mapping, modding and the game/community as a whole.
+
+If you have an idea for a topic to be discussed in the blog, feel free to contact any member of the QAT!
+
+## Current contributors
+
+- ![][flag_US] [Blocko](https://osu.ppy.sh/u/4075092)
+- ![][flag_US] [Cryptic](https://osu.ppy.sh/u/3728904)
```

Comment:
Cryptic, Feerum, and Ono aren't on the team

Reply:
likewise there are some QAT members that aren't on this list. not sure if they're meant to be perfectly overlapping